docs/cookbook/api.rst

```diff
@@ -0,0 +1,106 @@
+How to add a custom model?
+==========================
+
+In some cases you may be needing to add new models to your application in order to cover unique business needs.
+The proccess of extending Sylius with new entities is simple and intuitive.
+
+As an example we will take a **Supplier entity**, which may be really useful for shop maintenance.
+
+Authenitication
+---------------
+
+Creating OAuth client:
+
+.. code-block:: bash
+
+    $ php app/console sylius:oauth-server:create-client --grant-type="password" --grant-type="refresh_token" --grant-type="token"
+
+It will give you such a response:
+
+.. code-block:: bash
+
+    A new client with public id XYZ, secret ABC has been added
+
+Run your application on a built-in server:
+
+.. code-block:: bash
+
+    $ php app/console server:start localhost:8000
```

Comment:
Well... actually, not. It's a guide for greenies, they need to know where do they start the server.